acestep/text_tasks/external_lm_providers.py

```
@@ -53,6 +53,36 @@ class ExternalProviderProfile:
+            ("Mainland China chat completions", "https://api.minimaxi.com/v1/chat/completions"),
+        ),
+    ),
+    "minimax_anthropic": ExternalProviderProfile(
+        provider_id="minimax_anthropic",
+        label="MiniMax (Anthropic API)",
+        protocol="anthropic_messages",
+        default_model="MiniMax-M3",
+        default_base_url="https://api.minimax.io/anthropic/v1/messages",
+        api_key_env="ACESTEP_MINIMAX_API_KEY",
+        api_key_required=True,
+        secret_path_env="ACESTEP_MINIMAX_SECRET_PATH",
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "secret_path_env"

(S106)